ras_party/error_handlers.py

```diff
@@ -45,15 +27,15 @@ def http_error(error):
         response.status_code = error.response.status_code
     else:
         response.status_code = 500
-    logger.exception('Uncaught exception',
+    logger.exception('Error requesting another service',
                      url=request.url,
                      errors=errors,
                      status=response.status_code)
     return response
 
 
 @blueprint.app_errorhandler(Exception)
-def exception_error(_):
+def exception_error(e):
     logger.exception('Uncaught exception', status=500)
```

Comment:
I think we need to question whether this exception handler is giving us any value, right now it's just catching it and doing nothing with it.